### PR TITLE
[SYCL] change level0 to level_zero.

### PR DESCRIPTION
**Pull request by rbegam**
Signed-off-by: rbegam <rehana.begam@intel.com>

### CODE REVIEW

**Comment by rbegam** [comment]
please approve @bader

**Comment by bader** [comment]
> please approve @bader

I can do this, but in order to merge this PR code owners must approve as well. In this particular case runtime code owners must approve.

**Comment by rbegam** [comment]
ping @alexbatashev

**Comment by bader** [comment]
@rbegam, please, fix regressions in pre-commit testing.

**Comment by bader** [comment]
@rbegam, ping.

**Comment by romanovvlad** [comment]
Closing due to lack of activity. Please, feel free to reopen.

File: sycl/test/basic_tests/event.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// REQUIRES: opencl || level0
```
Comment (bader) [inline sycl/test/basic_tests/event.cpp:1]: Please, remove "|| level0".

**Comment by rbegam** [comment]
I am submitting a new PR related to this lit-test.

**Pull request closed without merging**